pom.xml

```diff
@@ -5,13 +5,13 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.0.RELEASE</version>
+		<version>2.2.4.RELEASE</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
-	<groupId>sumindaaa</groupId>
-	<artifactId>smart-employee-service</artifactId>
+	<groupId>com.sumindaa.demo</groupId>
+	<artifactId>employee-service</artifactId>
```

Comment:
I think smart-employee-service is the correct artifactid

Reply:
Can yo correct this?

Reply:
I think I done it correctly now